### ISSUE TITLE
Training with additional Batch Normalization layer producing strange results

### ISSUE THREAD

**Issue by immanuelazn** [Illumina/SpliceAI#131]
Hello,  

I am currently trying to reproduce the training of the 10k CNN, with tensorflow 2.8, as well as tensorflow 1.15.  I had added the batch normalization layer prior to the last layer, as described in #89 and #82 . 

During training of the technical replicates of the 10k model on the canonical, I am noticing very strange behaviour.  Firstly, three of the models reached decent metrics halfway through training.  One of the models reached a AUPRC of 0.9507 on validation, and 0.9451 on training by epoch 3.  Following however, there was a gradual decline in metrics until AUPRC reached 2e-3 for both training and validation by the final epoch.  I noticed the same issue on another model, with 0.8683 on validation, and 0.8961 on training.  The model gradually declined to 0 AUPRC on both training and validation, before gradually improving AUPRC for acceptor to ~0.49.  The same case happened with another model.  Final ensemble model performance has low AUPRC too, with 0.7907 and 0.83371 for acceptor and donor respectively.  

I have batch size set to 12, running on a single GPU.  Otherwise, no changes have been made, and I have attached the architecture below.  Any idea why this would occur?

[SpliceAI10000.txt](https://github.com/Illumina/SpliceAI/files/12102224/SpliceAI10000.txt)
[SpliceAI10000_c1.txt](https://github.com/Illumina/SpliceAI/files/12102225/SpliceAI10000_c1.txt)
[SpliceAI10000_c2.txt](https://github.com/Illumina/SpliceAI/files/12102220/SpliceAI10000_c2.txt)
[SpliceAI10000_c3.txt](https://github.com/Illumina/SpliceAI/files/12102221/SpliceAI10000_c3.txt)
[SpliceAI10000_c4.txt](https://github.com/Illumina/SpliceAI/files/12102222/SpliceAI10000_c4.txt)
[SpliceAI10000_c5.txt](https://github.com/Illumina/SpliceAI/files/12102223/SpliceAI10000_c5.txt)

![model](https://github.com/Illumina/SpliceAI/assets/56730419/010cb31b-6081-44fb-8cad-c6f4ca31d255)

**Comment by immanuelazn** [comment]
I've found that switching batch size to a static 6 has provided the results shown in the paper.
